README.md

```diff
@@ -1154,6 +1143,43 @@ workflow and time to run the activity attempt respectively), but the other spans
 are created in workflows and closed immediately since long-lived spans cannot work for durable software that may resume
 on other machines.
 
+### Rails
+
+Temporal Ruby SDK is a generic Ruby library that can work in any Ruby environment. However, there are some common
+conventions for Rails users to be aware of.
+
+See the [rails_app](https://github.com/temporalio/samples-ruby/tree/main/rails_app) sample for an example of using
+Temporal from Rails.
+
+#### ActiveRecord
+
+For ActiveRecord, or other general/ORM models that are used for a different purpose, it is not recommended to
+try to reuse them as Temporal models. Eventually model purposes diverge and models for a Temporal workflows/activities
+should be specific to their use for clarity and compatibility reasons. Also many Ruby ORMs do many lazy things and
+therefore provide unclear serialization semantics. Instead, consider having models specific for workflows/activities and
+translate to/from existing models as needed. See the [ActiveModel](#activemodel) section on how to do this with
+ActiveModel objects.
```

Comment:
Looks like a copy of what's above? Does it need to be in both spots?

Reply:
I intentionally put in both spots because I know some looking at data conversion may not look at rails and vice versa. Hopefully there's a future where docs.temporal.io obviates all of this.